pluma/stream/zeromq.py

```diff
@@ -3,22 +3,22 @@
 
 from pluma.stream import Stream, StreamType
 from pluma.stream.siconversion import SiUnitConversion
-from pluma.io.glia import load_glia
+from pluma.io.zeromq import load_zeromq
 from pluma.sync import ClockRefId
-from pluma.export.streams import export_stream_to_csv
 
 
-class GliaStream(Stream):
-    def __init__(self,
-                 filename: str,
+class ZeromqStream(Stream):
+    def __init__(self, 
+                 filenames: str, 
                  dtypes: list[list[tuple[str, type]]],
-                 data: pd.DataFrame = None,
+                 data: pd.DataFrame = None, 
```

Comment:
These seem like minor formatting changes only, in case you want to clean them. In a future PR we should probably add black as a dev dependency and run it before pushing code. No need to do it right now as it would probably revamp the entire code base :)